### DISCUSSION TITLE
still with pushkin version 1.1.5 after upgrade

### DISCUSSION THREAD

**Discussion by tmintz** [l3atbc-datadog/pushkin#134] [General] [answered]
Hi.  I ran `yarn global upgrade pushkin-cli`.  It ran (with two warnings:
```
[1/4] 🔍  Resolving packages...
warning pushkin-cli > knex > liftoff > findup-sync > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning pushkin-cli > knex > liftoff > findup-sync > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
```
But `pushkin --version` still says 1.1.5.
I did successfully upgrade from 1.1.4 earlier.

**Comment by parkerrobbins** [comment, marked as answer]
Hi, @tmintz! The warnings are normal. For some reason, to successfully upgrade you need the `--latest` flag. Try this: `yarn global upgrade pushkin-cli --latest`.

**Reply by tmintz** [reply]
that did it. Thanks!

**Reply by parkerrobbins** [reply]
No problem!
